include/zephyr/bluetooth/audio/audio.h

```diff
@@ -249,6 +250,7 @@ enum bt_audio_location {
 	BT_AUDIO_LOCATION_FRONT_RIGHT_WIDE = BIT(25),
 	BT_AUDIO_LOCATION_LEFT_SURROUND = BIT(26),
 	BT_AUDIO_LOCATION_RIGHT_SURROUND = BIT(27),
+	BT_AUDIO_LOCATION_RFU_START = BIT(28),
```

Comment:
I am a bit unsure about adding these RFU values to the `enum`. 

Before approving/disapproving it, I'd like input from @MariuszSkamra @alwa-nordic @jhedberg to whether or not we should indeed put RFU values in our enums that reflect Bluetooth Assigned Numbers. 

Ideally, to handle updates to the values, the verification of these values should be done by the application. This is, however, currently not possible with VOCS, which is why the check is currently implemented in the stack.

Reply:
@MariuszSkamra @sjanc Could i get your input on that?

Reply:
I agree with @Thalley that we should not put RFU values into `enum`

Reply:
Done